src/internal/packager/images/push.go

```diff
@@ -52,6 +53,11 @@ func (i *ImgConfig) PushToZarfRegistry() error {
 	spinner := message.NewProgressSpinner("Storing images in the zarf registry")
 	defer spinner.Stop()
 
+	if message.GetLogLevel() >= message.DebugLevel {
+		logs.Warn.SetOutput(spinner)
```

Comment:
```suggestion
	if message.GetLogLevel() >= message.DebugLevel {
		spinner.EnablePreserveWrites()
		logs.Warn.SetOutput(spinner)
```

If possible, can you also add this line in `src/internal/packager/images/pull.go` as well?  Looking at your code made me realize in many scenarios we are discarding this information when the spinner is destroyed.